### DISCUSSION TITLE
Feb 17

### DISCUSSION THREAD

**Discussion by oauch** [GymHubCommunity/GymHub-FE#172] [DailyMeeting]
### 개발 진행 상황
슬희: CORS 에러?! 해결 해야된다
수: 팔로우 기능 완료 / 메인 페이지 비어있을 때 디자인 완료
수영: preSigned URL 저장까지 완료
윤혁: 자잘한 디자인 수정 / 추가

### 지금 내 상태
슬희: 정신 없다 😮‍💨
수: 아는 맛 굿굿👍🏻
수영: preSigned URL 너무 빡셈.... 인생 쓰다...
윤혁: 잘하는지 뭐하는건지 모르겠다..

### 🏀 오늘 할 일

슬희: 백엔드분들이랑 CORS 해결하고 회원가입 완료 예정
수: 운동 선택 디자인
수영: 글 등록POST 요청
윤혁: 운동 추가 페이지

### 🎱 TMI

슬희: 월요일 수강신청
수: 포시가 바로 옆에 누워 있는데, 젤리를 만지면 스트레칭을 한다. 꼬순내 최고
수영: 하루 커피 2잔은 못 마신다.. 그래서 아침운동 하고 왔다
윤혁: 바지 샀던거 온다.. 긴팔티도 샀다..

**Comment by hanseulhee** [comment]
내 상태는 임티 머에여
